languages/fsharp/reference/exercise-concepts/meetup.md

```diff
@@ -0,0 +1,34 @@
+# Concepts of meetup
+
```

Comment:
Can you add a link to the v2 example implementation for reference please?

Reply:
Done!